src/pages/plan/hooks/useSocketHandler.tsx

```
@@ -0,0 +1,113 @@
+import { Client } from '@stomp/stompjs';
+import { useEffect, useMemo } from 'react';
+import StompURL from '../utils/StompURL';
+import { WaypointDispatcherResolver } from '../utils/WaypointDispatcherResolver';
+import { MemoDispatcherResolver } from '../utils/MemoDispatcherResolver';
+import { initDependencies } from '../utils/InitDependencies';
+import { topologicalSort } from '../utils/Topology';
+
+const API_BASE_URL = (import.meta as any).env?.VITE_API_BASE_URL || '/api';
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)